### DISCUSSION TITLE
Reversed Motor Direction Option (Props Out)

### DISCUSSION THREAD

**Discussion by mhk1058** [iNavFlight/inav-configurator#2027] [General]
I can't for the life of me get this to work.. what am I missing? Motors run just fine(Xing2 2207), everything else in the set up appears to be OK and runs as expected, SpeedyBee F405 V4 stack updated to latest firmware. Clicking the option and save, the option remains selected after reboot but the motors stay running original props in?

**Comment by sensei-hacker** [comment]
The option is telling INAV which direction the motors actually spin.  It's not to change which direction.

To change which direction, you can either swap any two motor wires, or reverse it in the ESC.

**Reply by mhk1058** [reply]
Aha... thanks, that now makes total sense!
